database/migrations/2018_06_22_111322_create_posts_db.php

```diff
@@ -33,6 +33,7 @@ public function up()
             $table->increments('id');
             $table->string('title')->nullable();
             $table->string('color')->nullable();
+            $table->string('featured')->nullable();
```

Comment:
You must not edit an already existing migration, create a new one instead with the changes!

Reply:
Created new migration.